### DISCUSSION TITLE
Pre-Start Check of Slider

### DISCUSSION THREAD

**Discussion by TGS78** [EdgeTX/edgetx#5486] [Q&A] [answered]
Hello,
I want to pre start check if sliders are in -100 position, but I always have to center both sliders.
Acc. to the EdgeTX knowledgebase it should probably work, but the "Man" setting is not shown, only On, Off, Auto.

Any ideas what I'm doing wrong? 
RC is a Jumper T20 V2

Pot warn. - When activated, this option checks the position of the pots & sliders. There are three options - OFF, Man and Auto. When ON or AUTO is selected, buttons for the available pots and sliders will appear. To enable the pot warning for an individual pot, select the pot with the [roller] or [dial] and click the button to highlight it. Highlighted pots are enabled.

Man - Positions are checked against manually configured pot and slider positions. To manually set the check position, select manual the from menu, select the item that you want to set, and long-press the [Enter] button to set its current position for the check.

**Comment by pfeerick** [comment, marked as answer]
The wording is wrong... will fix that in a minute. You are after ON, not Man. It must have changed from OFF, MAN, AUTO, to OFF, ON, AUTO at some point. Rest remains unchanged... i.e. set it to ON, then make sure both S3/S4 are active/ highlighted. Set them to the position you want them to be in, and then long press [Enter] on each to save that position.

**Reply by TGS78** [reply]
Works. Thanks a lot pfeerick!

**Reply by TGS78** [reply]
I recognized you've already updated the knowledgebase, I only suggest to delete the section marked below, it's confusing.

"ON - Positions are checked against manually configured pot and slider positions. To manually set the check position, **~~select manual the from menu,~~** select the item that you want to set, and long-press the [Enter] button to set its current position for the check."

**Reply by pfeerick** [reply]
Done. Thanks for pointing that out 👍
